services/src/main/java/org/exoplatform/task/rest/TaskRestService.java

```diff
@@ -587,13 +587,14 @@ public Response getTaskLogs(@ApiParam(value = "Task id", required = true) @PathP
   }
 
   @GET
-  @Path("comments/{id}")
+  @Path("comments/{id}/{lang}")
   @RolesAllowed("users")
   @Produces(MediaType.APPLICATION_JSON)
   @ApiOperation(value = "Gets a comment list of a specific task", httpMethod = "GET", response = Response.class, notes = "This returns a comment list of a specific task")
   @ApiResponses(value = { @ApiResponse(code = 200, message = "Request fulfilled"),
       @ApiResponse(code = 403, message = "Unauthorized operation"), @ApiResponse(code = 404, message = "Resource not found") })
   public Response getTaskComments(@ApiParam(value = "Task id", required = true) @PathParam("id") long id,
+                                  @ApiParam(value = "Lang", required = true) @PathParam("lang") String lang,
```

Comment:
here you should get the lang by the current  user with the conversation state, not by a parameter